deltachat-rpc-client/examples/echobot_no_hooks.py

```diff
@@ -40,13 +40,13 @@ def process_messages():
 
         while True:
             event = account.wait_for_event()
-            if event["type"] == EventType.INFO:
+            if event["kind"] == EventType.INFO:
                 logging.info("%s", event["msg"])
-            elif event["type"] == EventType.WARNING:
+            elif event["kind"] == EventType.WARNING:
                 logging.warning("%s", event["msg"])
-            elif event["type"] == EventType.ERROR:
+            elif event["kind"] == EventType.ERROR:
                 logging.error("%s", event["msg"])
-            elif event["type"] == EventType.INCOMING_MSG:
+            elif event["kind"] == EventType.INCOMING_MSG:
                 logging.info("Got an incoming message")
```

Comment:
this sequence is repeated in several places.  There should be a `account.wait_for_event(logging=True)` and/or a `account.wait_for_next_incoming_message(logging=true)` to have a standard convenient way to handle that. the `logging=True` means that core-events should map to `logging.*` .